templates/common/_base/units/machine-config-daemon-firstboot.service.yaml

```diff
@@ -8,7 +8,7 @@ contents: |
   # Removal of this file signals firstboot completion
   ConditionPathExists=/etc/ignition-machine-config-encapsulated.json
   After=machine-config-daemon-pull.service
-  Before=network-online.target crio.service kubelet.service ovs-configuration.service
```

Comment:
I think we want to keep ovs-configuration here, unless that dependency is just no longer needed for some other reason.

Reply:
Currently they're effectively exclusive I believe because of the `ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json`.

Reply:
That said...I think the dual-root upgrade model may push us away from the encapsulated stamp file.

So...a common systemd thing is to have a `foo.target` and `foo-pre.target`.  We could make all the kubelet deps be `After=kubelet-dependencies-pre.target`, and then stick `machine-config-daemon-firstboot.service` under `Before=kubelet-dependencies-pre.target` or so.

I don't think we need to block this PR on that though?

Reply:
wanted to double check. deps to ovs-configuration.service was added in recent PR https://github.com/openshift/machine-config-operator/pull/3858/files#diff-3380ef05804fcff21705fcca487ee27355893c239fb1f0df5077f9dc90cb6719R11 to keep things working when we skip reboot during initial pivot. Is this covered?

Reply:
OK yes you're right, this change will re-break that.